File: tags/pre-release-0.2/src/ConstraintSolver/IndependentDOF.cpp
#include "IndependentDOF.h"

using namespace std;
using namespace GiNaC;

IndependentDOF ::IndependentDOF ( double value, bool free):
DOF(free,false /*dependent*/)
{
	value_=value;
}

IndependentDOF :: IndependentDOF ( const char *name, double value, bool free):
DOF(name,free,false /*dependent*/)
{
	value_ = value;
}

File: trunk/src/mono/ConstraintSolver/IndependentDOF.cs

using System;

namespace pSketcher.ConstraintSolver
{

    /// <summary>
    /// The IndependentDOF class defines a DOF that depends on no other DOF's
    /// </summary>
    class IndependentDOF : DOF
    {

        public IndependentDOF (double value, bool free) : base (free)
        {
            this.Value = value;
        }

        //
        // Properties and accessor methods
        //
        public override double Value {get; set;}

    }
}
